### DISCUSSION TITLE
Getting task instance host is null while running the workflow

### DISCUSSION THREAD

**Discussion by aaryan-purohit** [apache/dolphinscheduler#16303] [General]
I have used docker-compose to start the apache dolphine scheduler server. I created a simple workflow, which contains 2 shell script with serial dependency. Now after saving and running the workflow, it is failing and while checking the logs, it is showing Task instance host is null as pop up. Can anyone help with this?

**Comment by Ishani-Gupta** [comment]
Were you able to fix this?
Facing same issue

**Reply by aaryan-purohit** [reply]
This is because of the memory issue. Try increasing the memory provided to different containers. That worked for me
